metadata_annotation_gdsc.ipynb

```diff
@@ -0,0 +1,2420 @@
+{
```

Comment:
<p>Wait. You can be annotating either the cell line OR the drug here, since it's from GDSC. So, for which drug is this <code>ln_ic50</code> from? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/pertpy-tutorials/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>